tests/verify-lvm-size.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
It would be ideal if we could use the lvs and vgs dicts from the setup facts for this. The only problem is that all of the sizes there are in GiB. This isn't a huge problem, and it might even be nice in the sense that it will eliminate failures based on tiny size differences. I wonder if there are other problems I'm not thinking of.

Reply:
Is the size variable going to get normalized to the same unit format at some point in the task flow (GiB, G, GB all become G)?

Reply:
See #27 for what size handling will look like. Presumably you could use `bsize` to parse (and later compare) the lvm sizes, whether from facts dict or cmdline tool, adding 'g' suffix if the former.